script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -20,7 +20,7 @@ The bathrooms in the mansion are huge, Arihiko. They're bigger than your room. A
 Thank you, Arihiko, for your encouraging response.
 It's a relief to hear something that I can agree with wholeheartedly.
 <Page6>
-"...Well, it can't be helped if we're talking about the mansion itself, though.
+"...Well, it can't be helped if we're talking about the mansion itself.
```

Comment:
I feel like I might be misinterpreting this line but I think he's saying like "from the building to the lifestyle, it's all the same"
> ...Well, even if we're talking about the mansion itself, that's just how it is